src/interp/interp.cc

```diff
@@ -208,6 +208,7 @@ Store::Store(const Features& features) : features_(features) {
   roots_.New(ref);
 }
 
+#ifndef NDEBUG
```

Comment:
Not sure I understand the purpose of wrapping this function in NDEBUG if it's only getting called inside `assert` anyway.

Reply:
Just so it clear to the reader that this is only needed/used in debug builds.

Reply:
:+1: